App.tsx

```diff
@@ -5,14 +5,31 @@ import ListScreen from './Screen/ListScreen';
 import FavoritesScreen from './Screen/FavoritesScreen';
 import {NavigationContainer} from '@react-navigation/native';
 import {createBottomTabNavigator} from '@react-navigation/bottom-tabs';
+import Icon from 'react-native-ionicons';
 
 const Tab = createBottomTabNavigator();
 
 export default class App extends React.Component {
   render() {
     return (
       <NavigationContainer>
-        <Tab.Navigator>
+        <Tab.Navigator
+          screenOptions={({route}) => ({
+            tabBarIcon: ({focused, color, size}) => {
+              let iconName;
+
+              if (route.name === 'Weather') {
+                iconName = focused ? 'star' : 'star-outline';
+              } else if (route.name === 'Favorites') {
+                iconName = focused ? 'heart' : 'heart-half';
+              }
```

Comment:
Try to use something here that would respect more the open close principle.
Advise : use an object that contains the route name as key, and an { focused : XX, default: YY } as value

Reply:
The idea is that if/when you add a new screen, you don't have to type code, but just to add a new entry to this "config" object